### PR TITLE
feat: Read-only connection string for PGD

### PR DESCRIPTION
**Pull request by vishalsawale9**
This PR intends to support the RO connection string & RO workloads for the PGD cluster through the terraform BA provider.

### CODE REVIEW

File: pkg/provider/resource_pgd.go [hunk 1]
```diff
@@ -331,6 +331,11 @@ func PgdSchema(ctx context.Context) schema.Schema {
 								},
 							},
 						},
+						"ro_connection_uri": schema.StringAttribute{
+							MarkdownDescription: "Cluster read-only connection URI. Only available for high availability clusters.",
```
Comment (wai-wong-edb) [inline pkg/provider/resource_pgd.go:335]: remove `Only available for high availability clusters`

File: pkg/provider/resource_pgd.go [hunk 2]
```diff
@@ -376,6 +381,11 @@ func PgdSchema(ctx context.Context) schema.Schema {
 							ElementType:   types.StringType,
 							PlanModifiers: []planmodifier.Set{setplanmodifier.UseStateForUnknown()},
 						},
+						"read_only_connections": schema.BoolAttribute{
+							Description: "Read-only connections.",
```
Comment (wai-wong-edb) [inline pkg/provider/resource_pgd.go:385]: Change to something like `Is read-only connections enabled.`

**Review by wai-wong-edb — COMMENTED** [review]
Requested changes

File: pkg/provider/resource_pgd.go (hunk 2 above)
Comment (vishalsawale9) [reply pkg/provider/resource_pgd.go]: Fixed in the latest commit.

File: pkg/provider/resource_pgd.go (hunk 1 above)
Comment (vishalsawale9) [reply pkg/provider/resource_pgd.go]: Fixed in the latest commit.

**Review by wai-wong-edb — APPROVED** [review]
LGTM

**Pull request merged**